Code/MinimalLib/common.h

```diff
@@ -388,6 +390,46 @@ std::string process_details(rj::Document &doc, const std::string &details,
     kekulize = true;
   }
 
+  const auto addChiralHsIt = doc.FindMember("addChiralHs");
+  if (addChiralHsIt != doc.MemberEnd()) {
+    if (!addChiralHsIt->value.IsBool()) {
+      return "JSON contains 'addChiralHs' field, but it is not a bool";
+    }
+    addChiralHs = addChiralHsIt->value.GetBool();
+  } else {
+    addChiralHs = true;
+  }
```

Comment:
it's not for this PR, but we really should do a `#define` for this pattern in some future refactoring exercise..

Reply:
Done. Nothing like writing a crappy preprocessor macro to make my day :-)